extensions-contrib/opentsdb-emitter/src/test/java/org/apache/druid/emitter/opentsdb/EventConverterTest.java

```diff
@@ -32,23 +32,29 @@
 
 public class EventConverterTest
 {
-  private EventConverter converter;
+  private EventConverter converterWithPrefix;
+  private EventConverter converterWithSpacePrefix;
+  private EventConverter converterWithoutPrefix;
 
   @Before
   public void setUp()
   {
-    converter = new EventConverter(new ObjectMapper(), null);
+    converterWithPrefix = new EventConverter(new ObjectMapper(), null, "druid");
+    converterWithSpacePrefix = new EventConverter(new ObjectMapper(), null, "legendary druid");
```

Comment:
Space prefix ? Maybe you should change `"legendary druid"` to `""`.

Reply:
You're right, i'll rename the variable to `converterWithPrefixContainingSpace`, and add another test for `converterWithPrefixContainingEmptyString` ("").
How does that sound?

Reply:
Sounds good.